api/applications_api/serializers.py

```diff
@@ -426,7 +426,7 @@ class ApplicationSerializer(serializers.ModelSerializer):
     contact_person = PersonSerializer(
         help_text="Contact person information for the application",
         read_only=False,
-        allow_null=True,
+        allow_null=False,
```

Comment:
I don't think we can set this to not allowing null, since the application is done in "steps", so  we don't have the possibility to add contact person until later. Probably need to validate this in validate_for_review

Reply:
is it so? I thought this would be initial when user wants to make the application

Reply:
I'll move this to `validate_for_review` :+1:

Reply:
Yeah, i think you save the first version of application with pretty much nothing